DatadogInternal/Sources/Telemetry/CoreLogger.swift

```diff
@@ -111,3 +111,12 @@ extension CoreLogger {
         log(.critical, message: message(), error: error)
     }
 }
+
+@objc
+public enum DDSDKVerbosityLevel: Int {
+    case none
+    case debug
+    case warn
+    case error
+    case critical
+}
```

Comment:
This should be part of `DatadogCore` IMO, and follow the new pattern with `objc_` and `@_spi`.